packaging/windows/build-win-installer.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import os
+import shutil
+import subprocess
+from pathlib import Path
+
+version = subprocess.run(
+    ["poetry", "version", "-s"], capture_output=True, text=True
+).stdout.rstrip()
+
+dir: Path = Path(__file__).resolve().parents[1]
+dist: Path = dir / "dist"
+Path(dist / "gaphor").mkdir(parents=True, exist_ok=True)
+
+gaphor_dist: Path = dist / "gaphor"
+seven_zip: Path = Path("C:/Program Files/7-Zip/7z.exe")
+payload: Path = dist / "payload.7z"
+portable: Path = dist / f"gaphor-{version}-portable"
+
+
+def clean_files() -> None:
```

Comment:
Can we add the paths (in this case `portable` and `payload`) as arguments to the function? That would make it more clear what will happen to whom.

Reply:
Great suggestion, I updated the function to pass in a list of paths as an argument.